src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="container-1" 
```

Comment:
Try to give more meaningful names for classes. And combine class names to combine styles in css
```suggestion
      class="input-container input-container--big" 
```